_posts/work/2018-01-30-hdf-in-the-cloud.md

```diff
@@ -64,40 +70,40 @@ So organizations like NASA are dumping large amounts of HDF onto Amazon's S3,
 that no one can actually read, except by downloading the entire file down to their hard drive,
 and then pulling out the particular bits that they need.
 This is inefficient.
-It misses out on the potential that cloud-hosted public data can offer our to society.
+It misses out on the potential that cloud-hosted public data can offer to our society.
 
 The rest of this post discusses a few of the options to solve this problem,
 including their advantages and disadvantages.
 
 1.  **Cloud Optimized GeoTIFF:** We can use modern and efficient formats from other domains, like Cloud Optimized GeoTIFF
 
-    **Good**: Fast, well established
+    *Good:* Fast, well established
 
-    **Bad**: Not actually sophisticated enough to handle some scientific use cases
+    *Bad:* Not actually sophisticated enough to handle some scientific use cases
 
```

Comment:
Cloud Optimized HDF5?  What would be needed to make this a reality?
My thoughts:
.         Use chunking for datasets larger than 1MB
·         Use “brick style” chunk layouts (enable slicing via any dimension)
·         Use readily available compression filters
·         Pack metadata in front of file (optimal for S3 VFD)
·         Provide info on chunk offsets
·         VLEN data in packed format (that I described last week)

These all seem quite possible without breaking backward compatibility or requiring a change in file format.